modules/react/select/stories/examples/Basic.tsx

```diff
@@ -1,21 +1,49 @@
 import React from 'react';
 import {FormField} from '@workday/canvas-kit-react/form-field';
-import {Select, SelectOption} from '@workday/canvas-kit-react/select';
+import {Select} from '@workday/canvas-kit-react/select';
+import {Flex} from '@workday/canvas-kit-react/layout';
+
+const options = [
+  {id: 'E-mail', data: {textValue: 'foo'}},
+  {id: 'Phone'},
+  {id: 'Fax (disabled)', disabled: true},
+  {id: 'Mail'},
+  {id: 'Mobile Phone'},
+  {
+    id: 'The Ontologically Anthropocentric Sensory Immersive Simulation',
+    disabled: false,
+  },
+];
```

Comment:
Where possible, can we use an array of strings instead of objects for our examples? Some examples will need to use objects (e.g., `Disabled Items` needs to use the `disabled` key), but many of the examples should work with simple strings. This also means we should be able to delete the disabled-related code (`disabledItems` and `nonInteractiveIds`) from most of our examples.

For example, I don't think the `Basic Example` needs to include any disabled items given the amount of code the disabled feature introduces. Best to limit the disabled code to the `Disabled Items` example.

Reply:
fair point